pom.xml

```diff
@@ -46,6 +46,7 @@
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <slf4j.version>1.7.36</slf4j.version>
+    <javaVersion>17</javaVersion>
```

Comment:
Servlet API 6.0.0 requires Java 11, not 17

Reply:
you are right, I'll adapt